Nantes/Nantes/ActionHandling/Link.swift

```diff
@@ -115,6 +115,13 @@ extension NantesLabel {
         return nil
     }
 
+	/// Checks if specified point contains a link
```

Comment:
Hey thanks for adding this.

What do you think about making https://github.com/instacart/Nantes/blob/ad2ce0367021e5afb2d5baa1775a65f56b4b1146/Nantes/Nantes/ActionHandling/Link.swift#L121 public and checking against nil instead? 

That way you get more information over just the boolean of existence or not, and since it uses the same code, it seems easier to just return the label we found if we found it.

Reply:
true that. made method public now and removed other method :)
09dbb28266e0ec7cd5999fb6842354f2a0d612a8